docs/how-to/publishing/android.rst

```diff
@@ -0,0 +1,132 @@
+.. Notes about this file:
+    The writing style may not match other BeeWare docs. I intend to revise it.
+    This file is a stub. Feedback very welcome, though.
+
+=======
+Android
+=======
+
+Overview
+--------
+
+The Google Play Store is the most widely-used Android app store. This guide
+focuses on how to distribute a BeeWare app there.
+
+Build the app in release mode
+-----------------------------
+
+Run this command::
+
+  briefcase publish android
+
+This will result in four APK files being generated. Each file corresponds
+to a different type of CPU that may be in an Android device, known as an
+Android ABI.
+
+.. TODO list:
+    - Adjust cookiecutter to make per-ABI APKs, per https://dev.to/sys1yagi/enable-split-apk-only-when-release-build-1ij
+    - Use Windows/Mac/Linux tabs for the above.
+    - Test it.
+
+
+Sign the APK files
+------------------
+
+.. admonition:: Create code signing identity
+
+  Before you sign the APK files, you need to :doc:`create a code signing
+  identity<../code-signing/android>`.
+
+Since you already have a code signing identity, we run the two Android SDK
+commands required to sign the APK files. First, we ensure the APK files are
+aligned to 32-bit boundaries, per the Android requirements.
+
+::
+
+  $ zipalign -p 4 -f filename.apk
+
+Next, we sign the actual package files.
+
+::
+
+  $ apksigner sign --ks my-release-key.jks my-app-unsigned-aligned.apk
+
+.. TODO list:
+    - Use Windows/Mac/Linux tabs for the above.
+    - Make sure we specify a path that finds these programs.
+    - Test it.
+    - Decide with Russell where we want to store these keys.
+
+Adding the app to the Google Play store
+---------------------------------------
+
+Adding the app requires a few steps. You need to have a Google Account and
+create a listing for your app. You will need to choose your app's signing
+configuration in the store. Finally, you will create a new release of the app,
+at which point you will upload the APK files that comprise your app.
+
+Creating a listing
+++++++++++++++++++
+
+To distribute your app on the Google Play Store, you need to visit
+https://play.google.com/apps/publish/ and create an account.
+
+Once you've done that, click **Create Application**. Choose a language and
+write a brief app title, up to 50 characters. This should probably be the
+same as your app's Formal Name in the BeeWare ecosystem.
+
+This will take you to **Store Listing** section of your app. You will need to
+answer a variety of questions. Take your time to answer them well.
+
+You will need to upload an icon. For briefcase apps, an icon is typically specified in
+your app's ``pyproject.toml`` file. Look for a line starting with ``icon = ``.
+The Google Play Store will require a variety of other information, including at least
+one screenshot of your app and your assessment of likelihood that children will want
+to use the app.
```

Comment:
Icon needs to be 512x512, which isn't a size required during setup.

Minium 2 screenshots are required; they must be at least 320px, no more than 3480px, and can't have an aspect ratio > 2:1.

Feature graphic (1024x500px) is also required.

Other details like a promo graphic, tv banner, promo vidio, and daydream 360 degree stereoscopic image can be provided, but aren't required.

Reply:
Fixed now, I believe.